themes/default/content/blog/aws-lambda-efs/index.md

```diff
@@ -7,8 +7,6 @@ authors:
     - luke-hoban
 tags:
     - AWS
-    - Lambda
```

Comment:
I think we still have "serverless" that would make sense here?

Reply:
Quite right. Fixed.